apis/apps/v1alpha1/cloneset_types.go

```diff
@@ -92,7 +92,7 @@ type CloneSetUpdateStrategy struct {
 	// Partition is the desired number of pods in old revisions. It means when partition
 	// is set during pods updating, (replicas - partition) number of pods will be updated.
 	// Default value is 0.
-	Partition *int32 `json:"partition,omitempty"`
+	Partition *intstr.IntOrString `json:"partition,omitempty"`
```

Comment:
You need to revise the comment accordingly since it is not a `int` type anymore.